examples/systemcalls/systemcalls.c

```diff
@@ -58,10 +78,42 @@ bool do_exec(int count, ...)
  *   as second argument to the execv() command.
  *   
 */
+    int status;								//status is passed into the waitpid function to get the staus of wait function
+    pid_t pid;								//stores the return value of fork function
+
+    //fork
+    pid = fork();
+
+
+    if(pid == -1)							//if the fork function returns -1 meaning the child process could not be created
+	    return -1;
```

Comment:
@Mich2899 pay attention to indent here and tabs/spaces, looks like your indent settings aren't consistent.